host-bin/enter-chroot

```diff
@@ -280,6 +280,45 @@ fi
 # Soft-link resolv.conf so that updates are automatically propagated
 ln -sf '/var/host/shill/resolv.conf' "$CHROOT/etc/resolv.conf"
 
+# Copy the current cmt driver and required files
+CHROMIUM_CMT=/var/lib/cmt
+if [ -L "$CHROOT/etc/crouton/cmt" ]; then
+  CHROOT_CMT=/etc/crouton/cmt
+
+  # xorg.conf.d cmt config
+  mkdir -p $CHROMIUM_CMT/etc/X11/xorg.conf.d
+  mkdir -p $CHROOT/etc/X11/xorg.conf.d
+  for xorgconf in /etc/X11/xorg.conf.d/*-touchpad-cmt.conf
+  do
+    xorgconffile=$(basename $xorgconf)
+    cp $xorgconf $CHROMIUM_CMT/etc/X11/xorg.conf.d
+    ln -sf $CHROOT_CMT/etc/X11/xorg.conf.d/$xorgconffile $CHROOT/etc/X11/xorg.conf.d/$xorgconfile
+  done
+
+  # cmt driver
+  mkdir -p $CHROMIUM_CMT/usr/lib/xorg/modules/input
+  cp /usr/lib64/xorg/modules/input/cmt_drv.so $CHROMIUM_CMT/usr/lib/xorg/modules/input
+  ln -sfT $CHROOT_CMT/usr/lib/xorg/modules/input/cmt_drv.so $CHROOT/usr/lib/xorg/modules/input/cmt_drv.so
+
+  # libs loaded by cmt driver
+  for lib in libevdev libgestures libjsoncpp
+  do
+    cp /usr/lib64/$lib.so $CHROMIUM_CMT/usr/lib
+    ln -sfT $CHROOT_CMT/usr/lib/$lib.so $CHROOT/usr/lib/$lib.so
```

Comment:
These are only required by the Xorg process, right?  Can you modify xserverrc-xorg to add an LD_LIBRARY_PATH instead of adding a bunch of symlinks to /usr/lib?